libtensor/CMakeLists.txt

```diff
@@ -285,7 +285,7 @@ set(SRC ${SRC_LINALG} ${SRC_KERNELS} ${SRC_GEN_BTOD} ${SRC_BTOD}
 )
 
 add_library(tensorlight ${SRC} ${SRC_CORE})
-target_include_directories(tensorlight PRIVATE "${libtensorlight_SOURCE_DIR}")
+target_include_directories(tensorlight PRIVATE "${libtensorlight_SOURCE_DIR}" PUBLIC "${BLAS_INCLUDE_DIR}")
```

Comment:
I'm not sure, but something to keep in mind: Perhaps this should be written to to `libtensorlight.pc` file.

Reply:
`-isystem @BLAS_INCLUDE_DIR@`?

Reply:
I can add this, but it's not tested... No problems until now with this, so probably we're good. I'll add a NOTE to the CMake though.

Reply:
Yes in theory, but I'm not convinced this is the right thing to do. I just wanted to point out that maybe. Effectively if this shoud be done depends if the blas headers are exposed to the public interface defined by libtensor via the headers. Now that might not be all that easy to find out unfortinately.

Reply:
Actually I think the answer is no. <cblas.h> is only use in compiled source units. Then the `target_include_directories` should even be PRIVATE.

Reply:
Yes, and `<mkl.h>` is only used in `BlasSequential.C`.

Reply:
Changing to `PRIVATE` 👍

Reply:
indeed.